### PR TITLE
Fix future and copy warning

### PR DESCRIPTION
**Pull request by markcyen**
## Pull request checklist
Please check if your PR fulfills the following requirements:
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Rubocop has passed locally and any fixes were made for failures
- [x] CircleCI build is passing
- [x] All tests passing locally

## Pull request type
Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [x] Other (please describe): the annoying warning fix on future and copy.

## What is the current behavior?
Displays warning when running test due to a future issue and setting with copy warnings 
Issue Number: 
closes #

## What is the new behavior?
No more warning when running test.
- 
- 
- 

## Does this introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

**Review by twarbelow — APPROVED** [review]
need to deep dive to see exactly what is going on. but seems to run fine :)

File: app/services/recommendation_service.rb [hunk 1]
```diff
@@ -16,6 +16,7 @@ def self.recommendation(user_id)
     baselines = merge_distances_and_ratings(cocktail_ratings, df, user_id)
     return nil if baselines.empty
     weighted_ratings(baselines, user_id)
+    weighted_rating = baselines[user_id] * baselines['value']
```
Comment (mkrumholz) [inline app/services/recommendation_service.rb:19]: Wait, @markcyen isn't this (l.19) doing the exact same thing as line 58 inside the weighted ratings method? We should probably remove line 18 if we want to do it this way, right?

File: app/services/recommendation_service.rb [hunk 2]
```diff
@@ -40,7 +41,7 @@ def self.euclidean_distance(df)
   end
 
   def self.user_distances(user_distance_matrix, df, user_id)
-    other_users = user_distance_matrix.loc[user_id].sort_values(0,ascending=true)
+    other_users = user_distance_matrix.loc[user_id].sort_values()
```
Comment (mkrumholz) [inline app/services/recommendation_service.rb:44]: 🎉

File: app/services/recommendation_service.rb (hunk 1 above)
Comment (markcyen) [reply app/services/recommendation_service.rb:19]: Ah yes, I must have uncommented it and meant to delete this line. Thanks for catching. Will push up

Comment (mkrumholz) [reply app/services/recommendation_service.rb]: Thanks! Looks good now

**Review by mkrumholz — APPROVED** [review]
Great work finding solutions here!

**Pull request merged**